diff --git a/Common/GPU/OpenGL/GLFeatures.cpp b/Common/GPU/OpenGL/GLFeatures.cpp
index 18aa1bd714..0843e38e78 100644
--- a/Common/GPU/OpenGL/GLFeatures.cpp
+++ b/Common/GPU/OpenGL/GLFeatures.cpp
@@ -400,7 +400,7 @@ bool CheckGLExtensions() {
 		gl_extensions.EXT_disjoint_timer_query = g_set_gl_extensions.count("GL_EXT_disjoint_timer_query") != 0;
 		gl_extensions.APPLE_clip_distance = g_set_gl_extensions.count("GL_APPLE_clip_distance") != 0;
 
-#if defined(__ANDROID__)
+#if 0
 		// On Android, incredibly, this is not consistently non-zero! It does seem to have the same value though.
 		// https://twitter.com/ID_AA_Carmack/status/387383037794603008
 #ifdef _DEBUG
@@ -453,7 +453,7 @@ bool CheckGLExtensions() {
 		gl_extensions.EXT_unpack_subimage = true;
 	}
 
-#if defined(__ANDROID__)
+#if 0
 	if (gl_extensions.OES_mapbuffer) {
 		glMapBuffer = (PFNGLMAPBUFFERPROC)eglGetProcAddress("glMapBufferOES");
 	}
diff --git a/Common/GPU/Vulkan/VulkanContext.cpp b/Common/GPU/Vulkan/VulkanContext.cpp
index 9a4e4c8fd4..82d53a8f6c 100644
--- a/Common/GPU/Vulkan/VulkanContext.cpp
+++ b/Common/GPU/Vulkan/VulkanContext.cpp
@@ -152,7 +152,7 @@ VkResult VulkanContext::CreateInstance(const CreateInfo &info) {
 	instance_extensions_enabled_.push_back(VK_KHR_SURFACE_EXTENSION_NAME);
 #ifdef _WIN32
 	instance_extensions_enabled_.push_back(VK_KHR_WIN32_SURFACE_EXTENSION_NAME);
-#elif defined(__ANDROID__)
+#elif 0
 	instance_extensions_enabled_.push_back(VK_KHR_ANDROID_SURFACE_EXTENSION_NAME);
 #else
 #if defined(VK_USE_PLATFORM_XLIB_KHR)
@@ -1060,7 +1060,7 @@ VkResult VulkanContext::ReinitSurface() {
 		break;
 	}
 #endif
-#if defined(__ANDROID__)
+#if 0
 	case WINDOWSYSTEM_ANDROID:
 	{
 		ANativeWindow *wnd = (ANativeWindow *)winsysData1_;
diff --git a/Common/GPU/Vulkan/VulkanLoader.cpp b/Common/GPU/Vulkan/VulkanLoader.cpp
index 9c6bde6be8..5b3658c45e 100644
--- a/Common/GPU/Vulkan/VulkanLoader.cpp
+++ b/Common/GPU/Vulkan/VulkanLoader.cpp
@@ -189,7 +189,7 @@ PFN_vkCmdCopyQueryPoolResults vkCmdCopyQueryPoolResults;
 PFN_vkCmdNextSubpass vkCmdNextSubpass;
 PFN_vkCmdExecuteCommands vkCmdExecuteCommands;
 
-#ifdef __ANDROID__
+#if 0
 PFN_vkCreateAndroidSurfaceKHR vkCreateAndroidSurfaceKHR;
 #elif defined(_WIN32)
 PFN_vkCreateWin32SurfaceKHR vkCreateWin32SurfaceKHR;
@@ -347,7 +347,7 @@ static const char * const so_names[] = {
 	"MoltenVK",
 #else
 	"libvulkan.so",
-#if !defined(__ANDROID__)
+#if 1
 	"libvulkan.so.1",
 #endif
 #endif
@@ -500,7 +500,7 @@ bool VulkanMayBeAvailable() {
 
 #ifdef _WIN32
 	const char * const platformSurfaceExtension = VK_KHR_WIN32_SURFACE_EXTENSION_NAME;
-#elif defined(__ANDROID__)
+#elif 0
 	const char *platformSurfaceExtension = VK_KHR_ANDROID_SURFACE_EXTENSION_NAME;
 #elif defined(VK_USE_PLATFORM_METAL_EXT)
 	const char * const platformSurfaceExtension = VK_EXT_METAL_SURFACE_EXTENSION_NAME;
@@ -749,7 +749,7 @@ void VulkanLoadInstanceFunctions(VkInstance instance, const VulkanExtensions &en
 
 #ifdef _WIN32
 	LOAD_INSTANCE_FUNC(instance, vkCreateWin32SurfaceKHR);
-#elif defined(__ANDROID__)
+#elif 0
 	LOAD_INSTANCE_FUNC(instance, vkCreateAndroidSurfaceKHR);
 #elif defined(VK_USE_PLATFORM_METAL_EXT)
 	LOAD_INSTANCE_FUNC(instance, vkCreateMetalSurfaceEXT);
diff --git a/Common/GPU/Vulkan/VulkanLoader.h b/Common/GPU/Vulkan/VulkanLoader.h
index d7709c0d86..9f13324672 100644
--- a/Common/GPU/Vulkan/VulkanLoader.h
+++ b/Common/GPU/Vulkan/VulkanLoader.h
@@ -191,7 +191,7 @@ extern PFN_vkCmdNextSubpass vkCmdNextSubpass;
 extern PFN_vkCmdEndRenderPass vkCmdEndRenderPass;
 extern PFN_vkCmdExecuteCommands vkCmdExecuteCommands;
 
-#ifdef __ANDROID__
+#if 0
 extern PFN_vkCreateAndroidSurfaceKHR vkCreateAndroidSurfaceKHR;
 #elif defined(_WIN32)
 extern PFN_vkCreateWin32SurfaceKHR vkCreateWin32SurfaceKHR;
diff --git a/Core/Instance.cpp b/Core/Instance.cpp
index fcb350f82e..864954a4c8 100644
--- a/Core/Instance.cpp
+++ b/Core/Instance.cpp
@@ -142,7 +142,7 @@ void InitInstanceCounter() {
 		PPSSPP_ID = 1;
 		return;
 	}
-#elif PPSSPP_PLATFORM(ANDROID) || defined(__LIBRETRO__) || PPSSPP_PLATFORM(SWITCH)
+#elif 1
 	// TODO : replace shm_open & shm_unlink with ashmem or android-shmem
 #else
 	// Create shared memory object
@@ -179,7 +179,7 @@ void ShutdownInstanceCounter() {
 		CloseHandle(mapLock);
 		mapLock = nullptr;
 	}
-#elif PPSSPP_PLATFORM(ANDROID) || defined(__LIBRETRO__) || PPSSPP_PLATFORM(SWITCH)
+#elif 1
 	// Do nothing
 #else
 	if (hIDMapFile >= 0) {
diff --git a/ppsspp_config.h b/ppsspp_config.h
index 7984a3d1b8..bc5f375157 100644
--- a/ppsspp_config.h
+++ b/ppsspp_config.h
@@ -117,7 +117,7 @@
     #endif
 #elif defined(__SWITCH__)
     #define PPSSPP_PLATFORM_SWITCH 1
-#elif defined(__ANDROID__)
+#elif 0
     #define PPSSPP_PLATFORM_ANDROID 1
     #define PPSSPP_PLATFORM_LINUX 1
 #elif defined(__linux__)
